### PR TITLE
Ansible certification

### PR DESCRIPTION
**Pull request by EduardoVV**

### CODE REVIEW

File: plugins/module_utils/policy_utils.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```
Comment (luispresuelVenafi) [inline plugins/module_utils/policy_utils.py:1]: This question applies for every other file that has this change. Why the change from "python3" to "python"?
it also seems you are giving backward compatibility to Python 2 because of the use of:
`from __future__ import`
We have dropped support for Python 2.

**Review by rvelaVenafi — APPROVED** [review]

**Review by marcos-albornoz — APPROVED** [review]
